packages/react/src/QuestionnaireForm/QuestionnaireForm.tsx

```diff
@@ -63,6 +66,8 @@ export function QuestionnaireForm(props: QuestionnaireFormProps): JSX.Element |
   const [response, setResponse] = useState<QuestionnaireResponse | undefined>();
   const [answers, setAnswers] = useState<Record<string, QuestionnaireResponseItemAnswer>>({});
   const [activePage, setActivePage] = useState(0);
+  const [questionnaireItems, setQuestionnaireItems] = useState(questionnaire?.item ?? []);
```

Comment:
I don't think I follow this... Why do we need to modify the questionnaire items?